genie-web/src/main/java/com/netflix/genie/web/tasks/leader/AgentJobCleanupTask.java

```diff
@@ -43,7 +43,8 @@
  */
 @Slf4j
 public class AgentJobCleanupTask extends LeaderTask {
-    private static final String STATUS_MESSAGE = "Agent AWOL for too long";
+    private static final String AWOL_STATUS_MESSAGE = "Agent AWOL for too long";
+    private static final String NEVER_CLAIMED_STATUS_MESSAGE = "No agent claimed the job for too long";
```

Comment:
General question. What happens if these jobs are marked failed and the agent comes up and tries to claim them?

Reply:
It will fail to claim and exit with non-zero exit code.